ci-operator/config/openshift/installer/openshift-installer-master__okd-scos.yaml

```diff
@@ -3,14 +3,22 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-9-golang-1.21-openshift-4.16
-  origin_scos-4.16_base-stream8:
+  origin_scos-4.16_base-stream9:
     name: scos-4.16
     namespace: origin
-    tag: base-stream8
-  origin_scos-4.16_installer-terraform-providers:
+    tag: base-stream9
+  origin_scos-4.16_etcd:
     name: scos-4.16
     namespace: origin
-    tag: installer-terraform-providers
+    tag: etcd
+  origin_scos-4.16_hyperkube:
+    name: scos-4.16
+    namespace: origin
+    tag: hyperkube
+  origin_scos-4.16_installer-terraform-providers-rhel9:
+    name: scos-4.16
+    namespace: origin
+    tag: installer-terraform-providers-rhel9
```

Comment:
does this exist?

Reply:
This does look like a miss at present. Working on it.